src/main/java/org/openrewrite/java/logging/slf4j/Slf4jLogShouldBeConstant.java

```diff
@@ -83,7 +83,7 @@ public J visitMethodInvocation(J.MethodInvocation method, ExecutionContext execu
                         if (STRING_FORMAT.matches(args.get(0))) {
                             J.MethodInvocation stringFormat = (J.MethodInvocation) args.get(0);
 
-                            if (stringFormat.getArguments().size() <= 1 ||
+                            if (stringFormat.getArguments().isEmpty() ||
```

Comment:
I'm not sure this would ever be empty, right? The compiler alone on String.format() requires at least one argument. 🤔 
Sorry to repeatedly bring up questions like this; really do appreciate the contribution, it's just that questions arise when you take a closer look; this is just as much an issue with the original implementation. I'm wondering if we could leave out the check completely, such that we now also rewrite for the tests that you've added.

Reply:
good point, the check is now redundant!

Don't worry, I really like the eye for detail! And if I keep adding new code I can expect new questions/comments.